portfolio/src/main/java/com/google/sps/servlets/WaypointsLoader.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.Grid;
+import com.google.sps.data.Waypoint;
+import java.io.FileReader;
+import java.io.BufferedReader; 
+import java.util.HashMap;
+import java.io.IOException;
+
+public class WaypointsLoader {
+  private static final int NUM_GRIDS_ROWS = 6;
+  private static final int NUM_GRIDS_COLS = 16;  
+  private static final HashMap<Grid, Waypoint> map = readWaypoints();
+  
+  public static HashMap<Grid,Waypoint> readWaypoints(){
+    try{
+    HashMap<Grid,Waypoint> map = new HashMap<>();
+    int gridRow = 0;
+    int gridCol = 0;
+    
+    BufferedReader waypointsReader = new BufferedReader(
+        new FileReader("/home/dpvalles/step148-2020/portfolio/waypoints.txt"));
```

Comment:
this is a hardcoded path that would only work on your computer. make this a relative path. move the waypoints.txt to java/com/google/sps/data/waypoints.txt and change this to just 'waypoints.txt' and it should work. 

if it doesn't work, then it's not worth spending too much time to figure it out. Just copy the contents of waypoints.txt into here directly. Ping me if that's unclear.

Reply:
Okay, I'll try to move it!